stripecardscan/src/main/java/com/stripe/android/stripecardscan/framework/api/StripeApi.kt

```diff
@@ -214,3 +216,71 @@ internal suspend fun uploadSavedFrames(
         errorSerializer = StripeServerErrorResponse.serializer(),
     )
 }
+
+internal fun isformatSupport(format: CardImageVerificationDetailsFormat) =
+    format == CardImageVerificationDetailsFormat.JPEG ||
+        format == CardImageVerificationDetailsFormat.WEBP
+
+internal fun imageWithConfig(
+    image: Bitmap,
+    format: CardImageVerificationDetailsFormat,
+    configs: CardImageVerificationDetailsAcceptedImageConfigs
```

Comment:
I'm a little wary of DTOs leaking into business logic. Can we create a new class here with similar values that's independent of the DTO?